Apps/Database/src/Delegates/DbBetaFeatureAccessDelegate.cs

```diff
@@ -48,9 +48,12 @@ public async Task DeleteRangeAsync(IEnumerable<BetaFeatureAccess> betaFeatureAcc
         }
 
         /// <inheritdoc/>
-        public async Task<IEnumerable<BetaFeatureAccess>> GetAllAsync(CancellationToken ct = default)
```

Comment:
GetAllAsync is needed to support this feature on the front-end:

![image](https://github.com/bcgov/healthgateway/assets/16570293/31375235-5f52-4b2e-b22b-072f0b930e7f)

Methods in the Controller & Service will need to be added for it as well.

Reply:
Added controller, service and delegate calls.